src/mappings/core.ts

```diff
@@ -314,11 +324,13 @@ export function handleSwap(event: SwapEvent): void {
   token0.volume = token0.volume.plus(amount0Abs)
   token0.totalValueLocked = token0.totalValueLocked.plus(amount0)
   token0.volumeUSD = token0.volumeUSD.plus(amountTotalUSD)
+  token0.feesUSD = token0.feesUSD.plus(feesUSD)
 
   // update token1 data
   token1.volume = token1.volume.plus(amount1Abs)
   token1.totalValueLocked = token1.totalValueLocked.plus(amount1)
   token1.volumeUSD = token1.volumeUSD.plus(amountTotalUSD)
+  token1.feesUSD = token1.feesUSD.plus(feesUSD)
```

Comment:
Need to confirm the logic around fees and tokens - im assuming it can be applied to both